enbeec/martins-aquarium
# Code Cleanup -- Mostly javascript organization

I originally wrote all the database->html functions in the same file. However, I think it is smart to split them out into their own files.

I did leave all of the source objects in the `database.js` with the functions that expose the data. My reasoning for this is that one day I might actually implement a real database and I can split the rest of the functionality out at that time.

## Steps to Test
1. Pull branch
2. Serve to browser
3. Verify that all sections still load
    - 2 fish: Bart, Geraldice
    - 3 quotes
    - 2 locations: Costa Rica, Jamaica
    - 3 tips
